po/fi.po

```diff
@@ -1,18 +1,19 @@
-# gnucash/fi.
+# Finnish translation of GnuCash.
 # This file is distributed under the same license as the gnucash package.
 # Copyright (C) 2008 gnucash project
+# Marko Kohtala <marko.kohtala@gmail.com>, 2020.
 # Tuomo Hartikainen <tth@harski.org>, 2017.
 # Tuomo Kohvakka <tuomo.kohvakka@iki.fi>, 2008.
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: GnuCash 2.6.15\n"
+"Project-Id-Version: GnuCash 3.10\n"
 "Report-Msgid-Bugs-To: https://bugs.gnucash.org/enter_bug.cgi?"
 "product=GnuCash&component=Translations\n"
 "POT-Creation-Date: 2020-03-28 12:48-0700\n"
 "PO-Revision-Date: 2017-03-13 20:59+0200\n"
-"Last-Translator: Tuomo Hartikainen <tth@harski.org>\n"
-"Language-Team: UNKNOWN\n"
+"Last-Translator: Marko Kohtala <marko.kohtala@gmail.com>\n"
+"Language-Team: Finnish <fi@li.org>\n"
```

Comment:
Is this the email address of a real language team like e.g. for the translation project translation-team-fi@lists.sourceforge.net?
If you are not associated to a team, insert "none" or leave it empty.

Reply:
This is just what gettext proposed. I'll change. I had done a quick search and found https://www.gnu.org/software/gettext/manual/gettext.html#Trans-Intro-0 mention it, but now looking it again there is a note the section is outdated.